cryptoki/README.md

```diff
@@ -7,4 +7,77 @@
 
 This is the high-level, Rust idiomatic wrapper crate for PKCS #11.
 
+The items in this crate only expose idiomatic and safe Rust types and
+functions to interface with the PKCS11 API. All the PKCS11 items might
+not be implemented but everything that is implemented is safe.
+
+## Example
+
+The following example initializes an empty token and generates a new RSA key.
```

Comment:
Any chance you could also add some comments throughout the code to broadly describe each section?

Reply:
Please see if this looks better. Frankly, I'm kind of bad with coming up with comments so if something's not good I'd be really glad if you added a couple of suggestions :bow:

Reply:
Awesome stuff, thank you!!